openedx_pearson_reports/api/v0/views.py

```diff
@@ -14,23 +14,23 @@
 from rest_framework.reverse import reverse
 from rest_framework.views import APIView
 
-from openedx_proversity_reports.edxapp_wrapper.openedx_authentication import \
+from openedx_pearson_reports.edxapp_wrapper.openedx_authentication import \
```

Comment:
A minor suggestion to consolidate code styling, either use:
```suggestion
from openedx_pearson_reports.edxapp_wrapper.openedx_authentication import (
    openedx_bearer_authentication
)
```
or
```suggestion
from openedx_pearson_reports.edxapp_wrapper.openedx_authentication import openedx_bearer_authentication
```

Reply:
Not the purpose of this RP, but applied... @diegomillan